.github/workflows/build.yaml

```diff
@@ -4,20 +4,45 @@ name: Build
 on:
   push:
     branches: [main]
+  workflow_call:
```

Comment:
Why is this needed?

Reply:
This lets the release.yaml workflow call this workflow when a release is tagged.

I didn't want to remove the on:push:main that you're already doing, but thought in the future that this would move to on:pr, and release would happen on push to main. 

The output here is leftover from an experiment and unneeded, I'll remove it.